cl_sii/dte/data_models.py

```diff
@@ -630,7 +630,12 @@ def __post_init__(self) -> None:
                 self.razon_ref)
 
 
-@dataclasses.dataclass(frozen=True)
+@pydantic.dataclasses.dataclass(
```

Comment:
Please mention in the commit message that this dataclass is now a Pydantic dataclass (instead of a std-lib one)

Reply:
Also, since this dataclass is now a Pydantic dataclass, I think `dc_field` shouldn't be used anymore. @jtrh ?

Reply:
> Also, since this dataclass is now a Pydantic dataclass, I think `dc_field` shouldn't be used anymore. @jtrh ?

`dc_field` (`dataclasses.field`) can be used both with Standard-Library Dataclasses and Pydantic Dataclasses. However, all the current uses of `dc_field` in `DteXmlData` are _already_ unnecessary, as `dc_field` is only required for some special cases, such as mutable default values (e.g. `mylist: List[int] = dc_field(default_factory=list)` instead of `mylist: List[int] = []`).

` = dc_field()` can be safely deleted, and ` = dc_field(default=None)` can be replaced with `= None`.

Related documentation:

- https://docs.python.org/3.7/library/dataclasses.html
- https://docs.python.org/3.7/library/dataclasses.html#dataclasses.field
- https://docs.python.org/3.7/library/dataclasses.html#init-only-variables